kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js

```diff
@@ -235,6 +235,7 @@ define([
             const cellCollapsed =
                     utils.getCellMeta(_cell, 'kbase.cellState.toggleMinMax', 'maximized') !==
                     'maximized',
+                // TODO: this needs to be updated for kbase.bulkImportCell
```

Comment:
Adding this comment in so it doesn't get forgotten. This is the text that appears on minimised cells to show the status. ATM we haven't got the FSM integrated into the bulk upload cell so no point in updating it yet.